soh/soh/Enhancements/randomizer/randomizer.cpp

```diff
@@ -1794,6 +1794,7 @@ void RandomizerSettingsWindow::DrawElement() {
                 mNeedsUpdate = true;
             }
             ImGui::PopStyleVar(1);
+            ImGui::EndDisabled();
```

Comment:
This should move before the `PopStyleVar` to maintain the call order of push/pop and begin/end

Reply:
TOOO LATE

Reply:
AHHHHHHH

Reply:
got a pr up for this at #3533 haha